### ISSUE TITLE
LICENSE file is MIT but HsSyck.cabal says BSD3

### ISSUE THREAD

**Issue by juhp** [audreyt/hssyck#4]
There seems to be discrepancy between the LICENSE file and
the license field in the .cabal file (MIT vs BSD3).
The current license file is not BSD.

cf http://opensource.org/licenses/category

(It seems C syck itself is BSD licensed.)

Could you please update the .cabal file to say MIT license, or
update the license file to be BSD if that is what is really intended,
for consistency and correctness? Thanks

(For context please see this Fedora package review:
<https://bugzilla.redhat.com/show_bug.cgi?id=1097035>.)

**Comment by mathstuf** [comment]
Ping? @audreyt

**Comment by audreyt** [comment]
That line pre-dates the time where Cabal had `MIT` as one of the values.

```
-- License is really "MIT", but Cabal doesn't have that constant yet
```

I've changed it to say `MIT` now.

**Comment by mathstuf** [comment]
Thanks. The tool gets the license from the .cabal file and we saw it in the RPM as mismatching, so that's why we missed the comment.

**Comment by audreyt** [comment]
No problem — Thanks for the catch! Released as 0.53.

**Comment by juhp** [comment]
Thank you very much, also for the updated release.
